controllers/suite_test.go

```diff
@@ -217,7 +217,7 @@ func setUpNamespace() {
 	}
 
 	err := k8sClient.Create(ctx, &testNamespace)
-	Expect(err == nil || errors.IsConflict(err))
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
This change doesn't LGTM because the test-suite should be able to be executed on an existing test namespace.

Reply:
+1